tests/acceptance/mocks/mock_dbus_server.py

```diff
@@ -21,29 +21,32 @@
 class IoMenderAuthenticationIface:
     """
 <node>
-	<interface name="io.mender.Authentication1">
-		<method name="GetJwtToken">
-			<arg type="s" name="token" direction="out"/>
-			<arg type="s" name="server_url" direction="out"/>
-		</method>
-		<method name="FetchJwtToken">
-			<arg type="b" name="success" direction="out"/>
-		</method>
+    <interface name="io.mender.Authentication1">
+        <method name="GetJwtToken">
+            <arg type="s" name="token" direction="out"/>
+            <arg type="s" name="server_url" direction="out"/>
+        </method>
+        <method name="FetchJwtToken">
+            <arg type="b" name="success" direction="out"/>
+        </method>
         <signal name="JwtTokenStateChange">
-			<arg type="s" name="token"/>
-			<arg type="s" name="server_url"/>
-		</signal>
-		<method name="MockSetJwtToken">
-			<arg type="s" name="token" direction="in"/>
-			<arg type="s" name="server_url" direction="in"/>
-		</method>
-		<method name="MockSetJwtTokenAndEmitSignal">
-			<arg type="s" name="token" direction="in"/>
-			<arg type="s" name="server_url" direction="in"/>
-		</method>
-	</interface>
+            <arg type="s" name="token"/>
+            <arg type="s" name="server_url"/>
+        </signal>
+        <method name="MockSetJwtToken">
+            <arg type="s" name="token" direction="in"/>
+            <arg type="s" name="server_url" direction="in"/>
+        </method>
+        <method name="MockSetJwtTokenAndEmitSignal">
+            <arg type="s" name="token" direction="in"/>
+            <arg type="s" name="server_url" direction="in"/>
+        </method>
+        <method name="MockEmitSignal">
+            <arg type="b" name="success" direction="out"/>
+        </method>
+    </interface>
 </node>
-	"""
+    """
```

Comment:
Oh sorry, the formatting test wouldn't pass unless i changed the formatting. I thought it may have been something with the formatter being updated. Is there any way I can come around the failing test without changing the formatting of the other parts?

Reply:
> Oh sorry, the formatting test wouldn't pass unless i changed the formatting. I thought it may have been something with the formatter being updated. Is there any way I can come around the failing test without changing the formatting of the other parts?

I am not aware of any recent change. We use a fixed version of the formatting tool.

But in any case you could have formatted the existing code first, commit that, then add your changes.

Again, this is not a big deal so don't spend more time on it.